chrome/content/test/tests.js

```diff
@@ -3,43 +3,7 @@
  * It uses the Jasmine JS testing library.
```

Comment:
> Rename this to match the name of the script being tested.

"this" as in the file ? It tests several scripts.

>  or label it as an integration test.

But it isn't an integration test.
What am I exactly supposed to do here ?